src/persist-client/src/lib.rs

```diff
@@ -63,7 +63,7 @@ pub use crate::r#impl::metrics::Metrics;
 ///
 /// This structure can be durably written down or transmitted for use by other
 /// processes. This location can contain any number of persist shards.
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
+#[derive(Arbitrary, Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
```

Comment:
does this mean we can use derive over manual impl for any other structs?

Reply:
We can, but I couldn't find any other serialized structs that were using `PersistClient` to change.

(there's `Controller`, but that's not going through the wire)